requirements.txt

```diff
@@ -14,6 +14,7 @@ graphene
 graphene-sqlalchemy
 jinja2
 markdown
+prometheus_client
```

Comment:
We also need this on the vagrant box --> https://github.com/fedora-infra/bodhi/blob/develop/devel/ansible/roles/bodhi/tasks/main.yml#L4

Reply:
Done!